deltachat-ffi/src/lib.rs

```diff
@@ -1041,6 +1041,23 @@ pub unsafe extern "C" fn dc_send_text_msg(
     })
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn dc_can_send_edit_request(
+    context: *mut dc_context_t,
+    msg_id: u32,
+    new_text: *const libc::c_char,
```

Comment:
There's no `new_text` param in the function signature in `deltachat.h`. Seems it's not really needed

Reply:
thanks, i added a commit!